specification/cognitiveservices/data-plane/Language/preview/2022-04-01-preview/textanalytics.json

```diff
@@ -2755,6 +3484,121 @@
       },
       "description": "if showStats=true was specified in the request this field will contain information about the document payload."
     },
+    "ConversationResult": {
+      "type": "object",
```

Comment:
At some point, can we expect descriptions on all models and properties? This contributes to both the REST API and SDK docs.

Reply:
I am in the process of adding descriptions for all new definitions for the AnalyzeConversations swagger.

Reply:
This will be added as part of a separate PR. Not blocking this PR for the definitions.